### PR TITLE
Changes for variant products, links in DQI tab and new languages

### PR DESCRIPTION
**Pull request by fleguellec**
Change in images, added new locales and variant product info following SaaS release

### CODE REVIEW

File: content/md/products-super-power/improve-data-quality.md [hunk 1]
```diff
@@ -58,17 +58,31 @@ To help you improve your products **enrichment** and **consistency**, simply fol
...
+Links can lead you either to a specific attribute or to the list of improvable attributes in the attribute panel.
+:::
+
+
+![Image data quality variant panel](../img/data-quality-variant-panel.png)
+
+For variant products, the data quality panel lists all attributes that need improvements to reach a higher data quality grade, whether at model or variant level.
+
+::: info
+Black attribute labels are not clickable and must be improved at upper level.
```
Comment (KmilleF) [inline content/md/products-super-power/improve-data-quality.md]: Attributes based at product model level, am I right?

Comment (fleguellec) [reply content/md/products-super-power/improve-data-quality.md]: yes: at product or sub-product. Should I word it differently?

File: content/md/products-super-power/improve-data-quality.md [hunk 2]
```diff
@@ -58,17 +58,31 @@ To help you improve your products **enrichment** and **consistency**, simply fol
...
 From this panel, you will know which attributes need to be improved and how to improve them.
 
+::: info
+Links can lead you either to a specific attribute or to the list of improvable attributes in the attribute panel.
+:::
+
+
+![Image data quality variant panel](../img/data-quality-variant-panel.png)
+
+For variant products, the data quality panel lists all attributes that need improvements to reach a higher data quality grade, whether at model or variant level.
```
Comment (camillebourjade) [inline content/md/products-super-power/improve-data-quality.md]:
```suggestion
For variant products, the data quality panel lists all the attributes that need improvements to reach a higher data quality grade, whether at model or variant level.
```

File: content/md/products-super-power/improve-data-quality.md (hunk 1 above)
Comment (camillebourjade) [reply content/md/products-super-power/improve-data-quality.md]: I would rephrase it, 'at upper level' isn't as clear as it could be.

File: content/md/products-super-power/improve-data-quality.md [hunk 3]
```diff
@@ -58,17 +58,31 @@ To help you improve your products **enrichment** and **consistency**, simply fol
...
+
+::: info
+Black attribute labels are not clickable and must be improved at upper level.
+Purple attribute labels are clickable and can be improved at variant level.
```
Comment (camillebourjade) [inline content/md/products-super-power/improve-data-quality.md]:
```suggestion
Purple attribute labels are clickable and can be improved at the variant level.
```

File: content/md/products-super-power/improve-data-quality.md [hunk 4]
```diff
@@ -58,17 +58,31 @@ To help you improve your products **enrichment** and **consistency**, simply fol
...
 ![Image data quality spellcheck corrections](../img/data-quality-spelling.png)
-![Image data quality title formatting suggestion](../img/data-quality-title-formatting.png)
 
 ::: info
 Even if they don't have permission to edit, all PIM users will be able to read data quality insights on all attributes.
 :::
 
 ::: info
-Spell check is available for English, French, German and Spanish locales.
+Spell check is available for English, French, German, Spanish, Italian, Dutch, Danish, Swedish locales.
```
Comment (camillebourjade) [inline content/md/products-super-power/improve-data-quality.md]:
```suggestion
Spell check is available for the English, French, German, Spanish, Italian, Dutch, Danish, and Swedish locales.
```

File: content/md/what-is/understand-data-quality.md [hunk 5]
```diff
@@ -35,8 +35,9 @@ The **consistency grade** is calculated on several criteria. The mix of the foll
 - Lowercases formatting: the PIM checks whether text area attributes are correctly formatted for lower cases.
 
 ::: info
-Spell check is available for English, French, German and Spanish locales.
+Spell check is available for English, French, German, Spanish, Italian, Dutch, Danish, Swedish locales.
```
Comment (camillebourjade) [inline content/md/what-is/understand-data-quality.md]:
```suggestion
Spell check is available for the English, French, German, Spanish, Italian, Dutch, Danish, and Swedish locales.
```

File: content/md/what-is/understand-data-quality.md [hunk 6]
```diff
@@ -35,8 +35,9 @@ The **consistency grade** is calculated on several criteria. The mix of the foll
...
 Spell check is usable with text and simple text area attributes.
+Spell check is usable with text, simple text area and rich text area attributes.
```
Comment (camillebourjade) [inline content/md/what-is/understand-data-quality.md]:
```suggestion
Spell check works on text, simple text area and rich text area attributes.
```

File: content/md/products-super-power/improve-data-quality.md (hunk 1 above)
Comment (fleguellec) [reply content/md/products-super-power/improve-data-quality.md]: done
